src/message_storage.py

```diff
@@ -3,21 +3,33 @@
 import os
 from dataclasses import dataclass, asdict
 
-import redis
-from dotenv import load_dotenv
 from redis import Redis
 from telegram import Update
 
-load_dotenv()
+from utils import str_to_bool
 
 logger = logging.getLogger(__name__)
 
 DEFAULT_MESSAGE_STORAGE = 100
 
-host = os.getenv('REDIS_HOST')
-port = os.getenv('REDIS_PORT')
-db = os.getenv('REDIS_DB')
-redis_client_singleton = redis.Redis(host=host, port=port, db=db)
+
+def configure_message_storage() -> bool:
```

Comment:
Focus: Created this to prevent it connecting to Redis before the main application started